store/src/pages/Buy.tsx

```diff
@@ -2,12 +2,14 @@ import React, { Component } from "react";
 
 import GridView from '../components/GridView';
 import DetailView from '../components/DetailView';
+import MelvinGridView from "../components/MelvinGridView";
```

Comment:
Please just work in the normal GridView component :)

Reply:
Yes I lost miself in github a bit, sorry :P